pkg/expr/nodes.go

```diff
@@ -227,6 +229,7 @@ func (dn *DSNode) Execute(ctx context.Context, now time.Time, _ mathexp.Vars, s
 		},
 		Headers: dn.request.Headers,
 	}
+	req.Headers[FromExpressionHeaderName] = "true"
```

Comment:
how do other datasources deal with this problem? It feels worth asking grafana plugins platform folks